deploy/osd-token-refresher/01-token-refresher.Deployment.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/component: authentication-proxy
+    app.kubernetes.io/name: token-refresher
+    app.kubernetes.io/version: master-2020-12-04-5504078
+  name: token-refresher
+  namespace: openshift-monitoring
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: authentication-proxy
+      app.kubernetes.io/name: token-refresher
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: authentication-proxy
+        app.kubernetes.io/name: token-refresher
+        app.kubernetes.io/version: master-2020-12-04-5504078
+    spec:
+      containers:
+      - args:
+        - --oidc.audience=observatorium-telemeter
```

Comment:
Is this the service account? in which case it should be observatorium-osd?

Reply:
No this is would be the [audience field](https://openid.net/specs/openid-connect-core-1_0.html#IDToken) for the requested OIDC token.